packages/calcite-components/src/components/flow-item/flow-item.tsx

```diff
@@ -50,7 +50,8 @@ import { CSS, ICONS, SLOTS } from "./resources";
  * @slot header-menu-actions - A slot for adding an overflow menu with `calcite-action`s inside a `calcite-dropdown`.
  * @slot fab - A slot for adding a `calcite-fab` (floating action button) to perform an action.
  * @slot footer-actions - [Deprecated] Use the `"footer"` slot instead. A slot for adding `calcite-button`s to the component's footer.
- * @slot footer - A slot for adding custom content to the component's footer.
```

Comment:
Why was the footer slot removed? This would be a breaking change.

Reply:
Sorry, I'll fix that.

Reply:
Can you also restore the doc for the `footer` slot?